src/applications/simple-forms/21-4138/helpers.js

```diff
@@ -61,3 +63,57 @@ export const isEligibleToSubmitStatement = formData => {
     isIneligibleForPriorityProcessing(formData)
   );
 };
+
+const upperizeKeys = obj => {
```

Comment:
if we're keeping this around, this would be great for a little unit test

Reply:
also I still kind of wonder if this is a backend bug. what do you think? if so, we could add a ticket to the backlog